src/pages/CompanyDetail/Component/HashTag.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './HashTag.scss';
+
+export default class HashTag extends React.Component {
+  render() {
+    return (
+      <article className="HashTag">
+        <ul>
+          {TAG_LIST.map(tags => {
```

Comment:
```suggestion
          {TAG_LIST.map(({id, tag})=> {
```
매개변수자리에서 바로 구조분해할당 할 수 있습니다!